core/src/main/java/com/sequenceiq/cloudbreak/service/stack/StackDtoService.java

```diff
@@ -102,7 +102,7 @@ public StackDto getByNameOrCrn(NameOrCrn nameOrCrn, String accountId, StackType
         StackView stackView = nameOrCrn.hasName() ?
                 getByName(accountId, nameOrCrn.getName(), stackType, config) :
                 getByCrn(nameOrCrn.getCrn(), stackType, config);
-        return getStackProxy(stackView, false);
+        return getStackProxy(stackView, true);
```

Comment:
do not change this, it affects performance of existing features

Reply:
done - added another endpoint which is essentially the same implementation except i have made the boolean value as a parameter. Only my endpoint calls it with 'true'. I need resources for my logic to work.